include/deal.II/lac/solver_cg.h

```diff
@@ -166,6 +166,24 @@ class SolverCG : public Solver<VECTOR>
          const VECTOR         &b,
          const PRECONDITIONER &precondition);
 
+  /**
+   * Connect a slot to retrieve the CG coefficients.
+   */
```

Comment:
also how often it will be called (only once at the end, or once per iteration?). same for the one below.